src/style.css

```diff
@@ -2,0 +1,61 @@
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir.ttc);
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-weight: bold;
+}
+
+.body {
+  margin: 0;
+}
+
+.form {
+  display: flex;
+  flex-direction: column;
+  margin: 20px 8px 0;
+}
+
+.form-field {
+  display: none;
+}
+
+.search {
+  font-weight: 300;
+  background-image: url(images/Icon.png);
+  background-repeat: no-repeat;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  outline: none;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  font-family: Avenir, sans-serif;
+}
+
+.search:focus {
+  font-weight: bold;
+}
```

Comment:
The `font-weight` property is being changed on focus. This can cause the layout to shift if the font weights render at different sizes, leading to a jarring user experience. It's usually best to avoid changing box-model properties (like width, height, padding, margin, border) or font properties that could affect the box size, on hover or focus states.